pkgs/development/python-modules/airlift/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, poetry-core, argparse, halo, pyyaml, hiyapyco, jinja2, dotmap, requests
+, kubernetes-helm, kind, docker, pythonOlder, buildPythonPackage
+, fetchFromGitHub }:
+
+buildPythonPackage rec {
+
+  disabled = pythonOlder "3.9";
+  pname = "airlift";
+  format = "pyproject";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "jl178";
+    repo = "airlift";
+    rev = "v${version}";
+    sha256 = "ZQ+YCMXQBM5Gb5O1jDQfKBeS1WxFKrtSRelgj65+f9U=";
+  };
+  nativeBuildInputs = [ poetry-core ];
+  propagatedBuildInputs = [
+    argparse
+    halo
+    pyyaml
+    hiyapyco
+    jinja2
+    dotmap
+    requests
+    kubernetes-helm
+    kind
+    docker
+  ];
```

Comment:
Could you please enable tests?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

Reply:
Tests are not currently a part of the package repo, but I can enable them once they are added upstream

Reply:
OK, then, could you please add `pythonImportsCheck` for a simple check?